distributed/deploy/local.py

```diff
@@ -0,0 +1,107 @@
+from __future__ import print_function, division, absolute_import
+
+from threading import Thread
+
+from tornado.ioloop import IOLoop
+from tornado.iostream import StreamClosedError
+from tornado import gen
+
+from ..utils import sync, ignoring, All
+from ..executor import Executor
+from ..nanny import Nanny
+from ..scheduler import Scheduler
+from ..worker import Worker, _ncores
+
+
+class Local(object):
+    def __init__(self, n_workers=None, cores_per_worker=None, processes=True, loop=None, start=True, scheduler_port=8786, **kwargs):
```

Comment:
Ok, I read too quickly. I am wondering if `threads_per_worker` would not be more explicit but then I assume we should use the "threads" terminology consistently throughout the code base and IIRC you intentionally switched from "threads" to "cores" in the `Executor` string representation in the past.

Maybe for the `processes` kwarg, it would be better to name it `use_nanny_processes` or just `used_nanny` to make that more explicit?

A docstring to explain the meaning of those parameters would help also :)